aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -5,17 +5,19 @@ description: Use .NET Core Generic Host in ASP.NET Core apps.  Generic Host is r
 monikerRange: '>= aspnetcore-3.1'
 ms.author: riande
 ms.custom: mvc
-ms.date: 4/17/2020
+ms.date: 11/09/2021
 no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: fundamentals/host/generic-host
 ---
 # .NET Generic Host in ASP.NET Core
 
 :::moniker range=">= aspnetcore-6.0"
 
-The ASP.NET Core templates create a .NET Core Generic Host (<xref:Microsoft.Extensions.Hosting.HostBuilder>).
+The ASP.NET Core templates create a <xref:Microsoft.AspNetCore.Builder.WebApplicationBuilder> and <xref:Microsoft.AspNetCore.Builder.WebApplication>, which configure and build a .NET Generic Host behind the scenes.
 
-This article provides information on using .NET Generic Host in ASP.NET Core. For information on using .NET Generic Host in console apps, see [.NET Generic Host](/dotnet/core/extensions/generic-host).
+For more information on configuring the .NET Generic Host using `WebApplicationBuilder`, see <xref:fundamentals/minimal-apis>. This article provides information on using the .NET Generic Host directly in ASP.NET Core.
```

Comment:
Do we need a strong message here to discourage using `Host`? If yes, how strong should it be? I know `WebApplication` is preferred, but does that mean this article is no longer relevant? What if people want to use the `Host` directly, or if they have to for anything `WebApplication` might not support?

Reply:
@halter73 can you provide some guidance on this topic?

Reply:
Our goal is to make `WebApplication` support any scenario where you would have used `            `Host.CreateDefaultBuilder(args) .ConfigureWebHostDefaults(webBuilder => ...` before. We believe this provides a better developer experience for configuring most web apps.

However, there are still some scenarios where using `Host` directly is more flexible particularly if you want non-default behavior. There's an issue to address this in .NET 7 at https://github.com/dotnet/aspnetcore/issues/32485.

I don't think we need a strong message discouraging using `Host`. It's not deprecated, and there are some scenarios where it's still better suited than `WebApplication`.